lms/djangoapps/course_goals/tests/test_api.py

```diff
@@ -32,21 +32,22 @@ def setUp(self):
         self.course = CourseFactory.create(emit_signals=True)
 
         self.user = User.objects.create_user('john', 'lennon@thebeatles.com', 'password')
-        CourseEnrollment.enroll(self.user, self.course.id)
+        self.enrollment = CourseEnrollment.enroll(self.user, self.course.id)
```

Comment:
I don't think you actually end up using this variable. But :shrug:

Reply:
Good catch. I added it as part of a test I thought we'd need, but removed that test.